README.md

```diff
@@ -45,7 +45,11 @@ This stateless service demostrates a watch dog application which is monitoring t
 For compiling the samples use gradle. This should be configured as part of the Service Fabric Java SDK installation.
 
 ## Deploying the samples
-All the samples once compiled can be deployed immediately using the install.sh scripts provided along with the sample. These scripts underneath uses azurecli. Before running the scripts you need to first connect to the cluster using azurecli. 
+All the samples once compiled can be deployed immediately using the install.sh scripts located inside Scripts folder of every sample. These scripts underneath uses azurecli. Before running the scripts you need to first connect to the cluster using azurecli. 
+
+## Compiling and Deploying the samples using Service Fabric Plugin for Eclipse
+The Service Fabric Plugin for Eclipse can be used to compile and deploy the samples, from Eclipse. To know more about this you can refer to our documentation - 

```

Comment:
I think its better if we capture eclipse and mac information in this readme itself.